spec/latest/json-ld-api/index.html

```diff
@@ -2786,31 +2784,56 @@ <h3>Algorithm</h3>
+                  append the values <code>@graph@index</code> and <code>@graph@index@set</code>
+                  to <em>containers</em>.</li>
+                <li>If the value does not contain the key <code>@id</code>,
+                  append the values <code>@graph@id</code> and <code>@graph@id@set</code>
+                  to <em>containers</em>.</li>
+                <li>Append the values <code>@index</code> and <code>@index@set</code>
+                  to <em>containers</em>.</li>
+              </ol>
+            </li>
               append <code>@graph</code>, <code>@id</code>, <code>@index</code> and <code>@set</code> to <em>containers</em>.</li>
             <li>Otherwise:
               <ol class="algorithm">
                 <li>If <em>value</em> is a <a>value object</a>:
                   <ol class="algorithm">
-                    <li class="changed">If <a data-link-for="JsonLdOptions">processingMode</a> is <code>json-ld-1.1</code>,
-                      append <code>@language</code> and <code>@index</code> to <em>containers</em>.</li>
                     <li>If <em>value</em> contains the key <code>@language</code>
                       and does not contain the key <code>@index</code>,
                       then set <em>type/language value</em> to its associated
-                      value and,
-                      <span class="changed">if <a data-link-for="JsonLdOptions">processingMode</a> is <code>json-ld-1.0</code></span>,
-                      append <code>@language</code> to
+                      value and, append <code>@language</code>
+                      <span class="changed">and <code>@language@set</code></span> to
                       <em>containers</em>.</li>
                     <li>Otherwise, if <em>value</em> contains the key
                       <code>@type</code>, then set <em>type/language value</em> to
                       its associated value and set <em>type/language</em> to
                       <code>@type</code>.</li>
+                    <li class="changed">Otherwise, if <em>value</em>
+                      does not contain the key <code>@index</code>, append <code>@language</code>
+                      and <code>@language@set</code> to
+                      <em>containers</em>.</li>
                   </ol>
                 </li>
                 <li>Otherwise, set <em>type/language</em> to <code>@type</code>
                   and set <em>type/language value</em> to <code>@id</code>,
-                  <span class="changed">and if <a data-link-for="JsonLdOptions">processingMode</a></span>
-                  is <code>json-ld-1.1</code>, append <code>@id</code>, <code>@type</code>,
-                  and <code>@index</code> to <em>containers</em>.</li>
+                  <span class="changed">and append <code>@id</code>, <code>@id@set</code>,
+                    <code>@type</code>, and <code>@type@set</code></span>,
```

Comment:
Should this be ordered as `@set@type`?  (Also, does that appear anywhere else in doc?)

Reply:
Ah, yes, lexicographically.